common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx

```diff
@@ -85,6 +86,7 @@ class AboutEventDisplay extends React.PureComponent<Props, State> {
       showPostRSVPModal: false,
       showCancelRSVPConfirmModal: false,
       startDate: startDate,
+      showHeaders: false,
```

Comment:
Where is this state value being used in the code?

Reply:
I have chaned the original state of showHeaders to have a boolean value. If I am not wrong this value will change to false on this key.